recipes/dcase2022_task4_baseline/README.md

```diff
@@ -32,55 +40,120 @@ The synthetic part of the dataset is composed of synthetically soundscapes, gene
 
 
+
 ## Training
-We provide three baselines for the task:
+We provide **three** baselines for the task:
 - SED baseline
-- baseline using pre-embeddings 
+- baseline using pre-trained embedding extractor DNN. 
 - baseline using Audioset data (real-world strong-label data)
 
-For now, only the SED baseline is available (the missing baseline will be published soon).
+For now, only the SED baseline is available (the missing baselines will be published soon).
 
-### SED Baseline
-The SED baseline can be run from scratch using the following command:
+### How to run the Baseline Systems
+The **SED baseline** can be run from scratch using the following command:
 
 `python train_sed.py`
 
+Note that the default training config will use 1 GPU. 
 Alternatively, we provide a [pre-trained checkpoint][zenodo_pretrained_models] along with tensorboard logs. The baseline can be tested on the development set of the dataset using the following command:
 
 `python train_sed.py --test_from_checkpoint /path/to/downloaded.ckpt`
 
-The tensorboard logs can be tested using the command `tensorboard --logdir="path/to/exp_folder"`. 
+The tensorboard logs can be tested using the command `tensorboard --logdir="path/to/exp_folder"`.
+
+
+## **(NEW!)** Energy Consumption
+
+In this year DCASE Task 4 Challenge, we also use energy consumption (kWh)
+via [CodeCarbon](https://github.com/mlco2/codecarbon) as an additional metric to rank the submitted systems.
+
+We encourage the participants to provide, for each submitted system (or at least the best one),
+using [CodeCarbon](https://github.com/mlco2/codecarbon), the following energy consumption figures in kWh:
+
+1) whole system training
+2) devtest inference
+3) evaluation set inference
+
+You can refer to [Codecarbon](https://github.com/mlco2/codecarbon) on how to do this (super simple!)
+or to this baseline code see `local/sed_trained.py` for some hints on 
+how we are doing this for the baseline system.
+
+
+**Important** 
+
+In addition to this, we kindly ask the participants to 
+run also, on the same hardware used for 2) and 3) 
+also energy consumption kWh for:
+
+1) devtest inference for baseline system using: 
+
+`python train_sed.py --test_from_checkpoint /path/to/downloaded.ckpt`
+You can find the energy consumed in kWh in `./exp/2022_baseline/devtest_codecarbon/devtest_tot_kwh.txt`
+
+2) evaluation set inference for baseline system using:
+`python train_sed.py --eval_from_checkpoint /path/to/downloaded.ckpt`
+You can find the energy consumed in kWh in `./exp/2022_baseline/evaluation_codecarbon/eval_tot_kwh.txt`
+
+**Why we require this ?**
+
+Energy consumption depends on hardware and each participant uses
+different hardware. 
+
+To obviate for this difference we use the baseline inference kWh energy consumption 
+as a common reference. Because of this is important that 
+inference energy consumption figures for both submitted system 
+and baseline are computed on same hardware under similar loading. 
```

Comment:
```suggestion
as a common reference. Because of this, it is important that the
inference energy consumption figures for both submitted system 
and baseline are computed on same hardware under similar loading. 
```